hycom/MSCPROGS/src/Nersclib/mod_hycomfile_io.F90

```diff
@@ -1182,13 +1182,13 @@ subroutine netcdfInfo(vnamein,gridrotate,stdname,longName,units,vname,cellmethod
          stdname='sea_water_salinity' ; units='1e-3' ; vname='so'
          limits=(/0,45/)
       case ('temp') 
-         stdname='sea_water_potential_temperature' ; units='Celsius' ; vname='theta0'
+         stdname='sea_water_potential_temperature' ; units='degrees C' ; vname='theta0'
```

Comment:
The Excel sheets states "thetao" instead of "theta0" as in theta ocean, I guess.

Reply:
yes I agree, that was a typo.

Reply:
Actually the CF checker gave an error for units='degrees C' as wite space is not accepted. So it  should be  units='degrees_C'. Now all fixed.